fronts-client/src/components/FrontsEdit/FrontSection/front-section.css

```diff
@@ -0,0 +1,17 @@
+/*
+	TODO: Move this logic into FrontSection.tsx when styledComponents is migrated to v6
+    styledComponents v6 will support container queries, which will allow us to remove this CSS file
+   */
+.front-header {
+	container-type: inline-size;
+}
```

Comment:
For those new to container queries: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries